packages/test-base/src/androidAndroidTest/kotlin/io/realm/kotlin/test/shared/RealmDictionaryTests.kt

```diff
@@ -489,6 +544,16 @@ internal interface DictionaryApiTester<T, Container> : ErrorCatcher {
     fun put()
     fun get()
     fun clear()
+    fun entries_size()
+    fun entries_add()
+    fun entries_addAll()
+    fun entries_clear()
+    fun entries_iteratorNext() // This tests also hasNext
+    fun entries_iteratorNext_managedEntry_setValue()
+    fun entries_iteratorRemove()
+    fun entries_iteratorConcurrentModification()
+    fun entries_remove()
+    fun entries_removeAll()
```

Comment:
It it s a bit hard to follow the logic in Github, but is `equals/hashcode` in the `MapEntry` implementations being tested anywhere in this?